WAW.API/Job/Domain/Models/Offer.cs

```diff
@@ -0,0 +1,12 @@
+using WAW.API.Shared.Domain.Model;
+
+namespace WAW.API.Job.Domain.Models;
+
+public class Offer : BaseModel {
+
+    public string? Title { get; set; }
+    public string? Image { get; set; }
+    public string? Description { get; set; }
+    public string? SalaryRange { get; set; }
```

Comment:
Title and SalaryRange shouldn't be nullable, as per our model the only optional properties are Image and Description. Assign `string.Empty` to those that're not nullable.